qiita_pet/handlers/api_proxy/studies.py

```diff
@@ -173,10 +173,14 @@ def study_prep_get_req(study_id, user_id):
                 info['start_artifact_id'] = start_artifact.id
                 info['youngest_artifact'] = '%s - %s' % (
                     youngest_artifact.name, youngest_artifact.artifact_type)
+                info['ebi_experiment'] = (True
```

Comment:
I think this line can be shortened to `bool(prep.ebi_experiment_accessions)`, at least that's what the `if` would do IIRC.